src/app/lib/config/services/serbian.js

```diff
@@ -259,18 +259,18 @@ export const service = {
       },
       socialEmbed: {
         caption: {
-          textPrefixVisuallyHidden: 'Video caption, ',
-          text: 'Warning: Third party content may contain adverts',
+          textPrefixVisuallyHidden: 'Potpis ispod videa, ',
+          text: 'Upozorenje: Sadržaj drugih sajtova može da sadrži i reklame',
         },
         fallback: {
-          text: 'Content is not available',
-          linkText: 'View content on %provider_name%',
-          linkTextSuffixVisuallyHidden: ', external',
+          text: 'Sadržaj nije dostupan',
+          linkText: 'Pogledajte više na %provider_name%',
+          linkTextSuffixVisuallyHidden: ', spoljna stranica',
           warningText: 'BBC nije odgovoran za sadržaj drugih sajtova.',
         },
         skipLink: {
-          text: 'Skip %provider_name% post',
-          endTextVisuallyHidden: 'End of %provider_name% post',
+          text: 'Preskočite sadržaj sa %provider_name%',
+          endTextVisuallyHidden: 'Kraj sadržaja sa [%provider_name%',
```

Comment:
Is that opening square bracket `[` intentional?

Reply:
@simonsinclair Let me fix this 👍